src/Templates/src/templates/maui-blazor/MauiApp.1.csproj

```diff
@@ -64,4 +64,8 @@
         <PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="MS_EXT_LOG_DEBUG_VERSION" />
     </ItemGroup>
 
+    <!-- Apple Privacy Manifest file, more information: https://aka.ms/maui-privacy-manifest -->
+    <ItemGroup Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'ios'">
+        <BundleResource Include="Platforms\iOS\PrivacyInfo.xcprivacy" LogicalName="PrivacyInfo.xcprivacy" />
+    </ItemGroup>
```

Comment:
Some thoughts here:
1. Why does this need to be in the project file at all? Can we do a default MSBuild wildcard instead? Seems like the presence of the file would be enough.

2. Why isn't `**\*.xcprivacy` in the iOS workload's default wildcards?

https://github.com/xamarin/xamarin-macios/blob/c32c925eb52239e1cef8b6e708bcbff7f015ec9f/dotnet/targets/Microsoft.Sdk.DefaultItems.template.props#L62-L79

If it were, I believe something in MAUI already sets `$(_ResourcePrefix)` to `Platforms\iOS`, so everything would "just work"?

/cc @rolfbjarne

Reply:
@jfversluis it looks like I got mixed up in `Include` vs `Exclude`. Is this actually working and you don't need the `.csproj` entry at all? Is `Platforms\iOS\PrivacyInfo.xcprivacy` being added twice to `@(BundleResource)` with these changes?

Reply:
MAUI might actually be using this:

https://github.com/dotnet/maui/blob/402fdf7fcc5a550ebc32512791a1121d1961c1bb/src/Controls/src/Build.Tasks/nuget/buildTransitive/netstandard2.0/Microsoft.Maui.Controls.Globs.props#L66-L70

So we could add logic here instead of the template if needed.

Reply:
What would happen if multiple nuget package libraries also included 
`<BundleResource Include="libs\net8.0-ios\PrivacyInfo.xcprivacy" LogicalName="PrivacyInfo.xcprivacy" />`
Wouldn't they start overwriting each other or get conflicts?

Reply:
I think the wildcard would only be meant for “app” projects. I don’t think there is a solution for NuGet packages yet. Something in the iOS workload would need to support merging these files. Is there an issue tracking this @rolfbjarne?

Reply:
Gotcha. I thought that might be the solution when you pointed out you can use * includes. The recent blogpost on this was very vague on what class libraries should do even though it clearly states class libraries should provide a privacy file.

Reply:
> Why does this need to be in the project file at all? Can we do a default MSBuild wildcard instead? Seems like the presence of the PrivacyInfo.xcprivacy file would be enough.

> Why isn't **\*.xcprivacy in the iOS workload's default wildcards?

I like the idea that this should be included by default.

> If it were, I believe something in MAUI already sets $(_ResourcePrefix) to Platforms\iOS, so everything would "just work"?

MAUI sets `$(IPhoneResourcePrefix)`, but not early enough: https://github.com/dotnet/maui/issues/16734#issuecomment-1677601531

> What would happen if multiple nuget package libraries also included
<BundleResource Include="libs\net8.0-ios\PrivacyInfo.xcprivacy" LogicalName="PrivacyInfo.xcprivacy" />
Wouldn't they start overwriting each other or get conflicts?

Native frameworks (.framework directories) must ship their own xcprivacy files (inside the .framework), so this (mostly) covers binding projects / NuGets.

Otherwise if a NuGet accesses APIs that need a mention in the privacy manifest, they must currently document it, and the app developer must manually add the corresponding entry in their own xcprivacy file.

I had an idea to handle this automatically where the app developer would get a build warning without the appropriate mention in the privacy file (https://github.com/xamarin/xamarin-macios/issues/20059#issuecomment-1969367180), but we decided to wait on that until at least the summer, to see if Apple comes up with something else for Xcode 16 / iOS 18.

Note that the you have to document the reason you're using a specific API in the privacy manifest, which only app developers (and not NuGet developers) know. Thus it doesn't make much sense to support merging privacy files from NuGets.

Reply:
> Thus it doesn't make much sense to support merging privacy files from NuGets.

My understanding is that's exactly what's happening with all the privacy files that are inside .xcframework files. So isn't that the same thing to have nugets do that?

Reply:
> > Thus it doesn't make much sense to support merging privacy files from NuGets.
> 
> My understanding is that's exactly what's happening with all the privacy files that are inside .xcframework files. So isn't that the same thing to have nugets do that?

No, the privacy files in xcframeworks stay inside the corresponding .framework when added to the app bundle, so each framework has its own privacy file.